compositions/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,49 @@
+import { BoxProps, Flex, Icon } from '@zoralabs/zord'
+
+import { useToggle } from '@shared/hooks'
+import clsx from 'clsx'
+import * as styles from './SearchInput.css'
+import useAutoFocus from '@shared/hooks/useAutoFocus'
+
+interface SearchInputProps extends BoxProps {
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+  placeholder?: string
+  inputClassNameOverrides?: string
+  visible?: boolean
+}
+
+export function SearchInput({
+  onChange,
+  className,
+  inputClassNameOverrides,
+  placeholder = 'Search Web3...',
+  visible = false,
```

Comment:
not really needed, since `useAutoFocus` also have a default

Reply:
Good callout